packages/oui-button/src/button.controller.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash'
+
+export default class {
+  constructor ($attrs, $element, $log) {
+    'ngInject'
+
+    this.$attrs = $attrs
+    this.$element = $element
+    this.$log = $log
+  }
+
+  $onInit () {
```

Comment:
Could we support also this syntax for the `disabled` attribute ? `<oui-button disabled></oui-button>`

I think it can be done with this:

```javascript
if(this.$attrs.hasOwnProperty('disabled') && this.$attrs.disabled === "") {
  this.disabled = true
}
```

Reply:
Yes :)

Reply:
I have a helper for that.